installers/ansible/roles/pgo-operator/tasks/namespace.yml

```diff
@@ -14,7 +14,7 @@
     target_ns_script: "{{ lookup('template', 'add-targeted-namespace.sh.j2') }}"
   with_items: "{{ nslist.split(',') | map('trim') | list }}"
   when:
-    - reconcile_rbac == 'false'
+    - not (reconcile_rbac | default(False) | bool)
```

Comment:
Isn't this defaulted to `true`? cc @andrewlecuyer

Reply:
yeah it is

Reply:
It is defaulted to `true` in the `pgo-operator/defaults`, do we need to have a default here too?

Reply:
@jmckulk I don't know -- what's the purpose of the `default(False)` here?

Reply:
@jkatz correct, `reconcile_rbac` should default to `true`.

Reply:
to set a default value when `reconcile_rbac` is not set. In this case it is always set because of the default defaults.yaml

Reply:
So do we need a `default` block?

Reply:
no it isn't getting used here. Ill remove it

Reply:
Thanks.